application-accelerator/vscode.hbs.md

```diff
@@ -86,10 +86,42 @@ NAME      FQDN                                      TLS SECRET     STATUS   STAT
 tap-gui   tap-gui.tap.tapdemo.myorg.com             tap-gui-cert   valid    Valid HTTPProxy
 ```
 
-## <a id="app-acc-known-issues"></a> Known Issues
```

Comment:
instead of removing the known issues section, this should be moved to a new section called `Resolved issues`

Reply:
I actually disagree with this - in the release notes, there is a section in there for resolved issues. I'd imagine this kind of information would be better suited to live there than within this doc. Just my 2 cents :)

Reply:
gotcha, I wasn't aware of that section in the release notes, I'll remove that section from the doc